freqtrade/exchange/binance.py

```diff
@@ -163,12 +182,14 @@ def get_max_leverage(self, pair: Optional[str], nominal_value: Optional[float])
         """
         if pair not in self._leverage_brackets:
             return 1.0
+        if (pair is None or nominal_value is None):
```

Comment:
I didn't find any instance where `get_max_leverage` would be called with optional pair/nominal values where they're not "something" (except for tests, maybe), so i think the correct approach is  to remove the `Optional` from the function signature (here and on the parent method) and remove this check (it's really pointless - as we control all methods that call exchange methods).